tools/build_script_generator/scons/resources/build_target.py.in

```diff
@@ -43,9 +44,11 @@ def build_target(env, sources):
 		%% if upload_with_artifact
 	env.Alias("program", [env.OpenOcd(program, commands=["modm_program {$SOURCE}"]), "artifact"])
 	env.Alias("bmp", [env.BlackMagicProbe(program), "artifact"])
+	env.Alias('dfu', [env.DfuStm32Programmer(env.Bin(program)), "artifact"])
 		%% else
 	env.Alias("program", env.OpenOcd(program, commands=["modm_program {$SOURCE}"]))
 	env.Alias("bmp", env.BlackMagicProbe(program))
+	env.Alias('dfu', env.DfuStm32Programmer(env.Bin(program)))
 		%% endif
 
 	%% elif core.startswith("avr")
```

Comment:
Please also update the `module.md` docs with the `scons dfu` addition, with simplified output, like the scons program one.